athf/mcp/tools/agent_tools.py

```
@@ -0,0 +1,151 @@
+
+        full_report = "\n".join(report_parts)
+
+        # Save research file
+        from athf.core.research_manager import ResearchManager
+
+        rm = ResearchManager(research_dir=workspace / "research")
+        rid = rm.get_next_research_id()
+
+        frontmatter = {
+            "research_id": rid,
+            "title": "{} Research".format(topic),
```

Ruff (0.15.6)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)